src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return super.toString();
+    }
```

Comment:
The `toString` method should return a string representation of the Ball object, including its color and number. Currently, it's returning the default `toString` representation from the Object class which is not informative. Consider providing a custom string format that includes the Ball's color and number.